mirascope/tools/requests.py

```diff
@@ -0,0 +1,48 @@
+from typing import ClassVar, Literal
+
+import requests
+from pydantic import Field
+
+from mirascope.tools.base import ConfigurableTool, _ToolConfig
+
+
+class RequestsConfig(_ToolConfig):
+    """Configuration for HTTP requests"""
+
+    timeout: int = Field(default=5, description="Request timeout in seconds")
+
+    @classmethod
+    def from_env(cls) -> "RequestsConfig":
+        return cls()
+
+
+class Requests(ConfigurableTool):
+    """Tool for making HTTP requests to fetch content from URLs."""
```

Comment:
this docstring will be inserted into the prompt as the tool's description, so it's important that we do a good job of making this clear while also minimizing tokens. The more clear this docstring, the better the tool will likely work with the LLM.

For example, you say `fetch content...` but the `method` field below includes POST, PUT, and DELETE, which seem like not just fetch? The language here needs to be clear so the LLM uses the tool properly, so just want to make sure this point is clear and we write really good docstrings / descriptions for our pre-made tools